### ISSUE TITLE
Config and commands don't align

### ISSUE THREAD

**Issue by silasb** [rubyatscale/packs#112]
The documentation around [configuration](https://github.com/rubyatscale/packs#configuration) mentions a way to override where to look for packs, but it seems like `packs create` doesn't honor this and only allows for creating packs in [certain folders](https://github.com/rubyatscale/packs/blob/88ab0c716364841396df6b4c7f99718f3d37788b/lib/packs.rb#L28-L32), not the one you override in the configuration.

I understand the configuration file syntax (regexp) doesn't make this easy to figure out where the source of the packs is located at.  Could the configuration `pack_paths`, just always assume nested packs and change from `regexp` to strings instead?

The default configure file would then be:

```
packs_paths:
  - gems
  - components
  - packs
```

and the configuration override example could be:

```
packs_path:
  - packages
  # or
  - modules/systems
```

Which would be the same contents as [PERMITTED_PACK_LOCATIONS](https://github.com/rubyatscale/packs/blob/88ab0c716364841396df6b4c7f99718f3d37788b/lib/packs.rb#L28-L32) today.  Moving forward `PERMITTED_PACK_LOCATIONS` would likely move to a method that evals the config or uses the default if the config isn't present.

This would allow the create command to just work™️:

```
./bin/packs create modules/systems/disco-service
```

**Comment by alexevanczuk** [comment]
@silasb Thanks for opening this issue! That command should definitely permit packs to be created in any location that matches `pack_paths`.

Are you interested in submitting a PR for this change? If you are, I'm happy to support and/or review!

Otherwise, this may be a good starter task @shageman if you can find anyone interested (let me know if not).

**Comment by diogogallo** [comment]
This has been fixed in v0.0.36 (https://github.com/rubyatscale/packs/pull/130). Tested locally and working fine
